src/Features/VolumetricLighting.h

```diff
@@ -21,18 +33,13 @@ struct VolumetricLighting : Feature
 	virtual inline std::string GetName() override { return "Volumetric Lighting"; }
 	virtual inline std::string GetShortName() override { return "VolumetricLighting"; }
 
-	virtual void Reset() override;
-
 	virtual void SaveSettings(json&) override;
 	virtual void LoadSettings(json&) override;
 	virtual void RestoreDefaultSettings() override;
 	virtual void DrawSettings() override;
 	virtual void DataLoaded() override;
 	virtual void PostPostLoad() override;
-
-	std::map<std::string, Util::GameSetting> VLSettings{
-		{ "iVolumetricLightingQuality:Display", { "Lighting Quality", "Adjusts the quality of volumetric lighting. [-1,-2] (off, low, mid, high).", REL::Relocate<uintptr_t>(0, 0, 0x1ed4030), 1, -1, 2 } },
```

Comment:
Are we completely ignoring any built in settings now? To the extent they function, we should try to leverage existing ini settings instead of creating completely new stuff. Of course, some are broken across versions so that would be a reason to create our own customer setting.

Reply:
Ok did a review without reading your post, so you can ignore if indeed broken.

Reply:
Yeah for AE/SE/VR it reads the default setting values, these are the ones loaded from the ini for AE/SE or the ones baked into the exe for VR. It caches the High setting values and overwrites it if custom settings are used.